File: backend/app/schemas/notification.py
"""消息通知 Schema（与前端 types/notification.ts 对齐）"""
from pydantic import BaseModel


class NotificationResponse(BaseModel):
    """消息记录（对齐前端 NotificationItem）"""
    id: int
    type: str
    title: str
    summary: str = ""
    content: str = ""
    status: str = "unread"
    created_at: str = ""

File: frontend/src/types/notification.ts
/** 通知类型 */
export const NotifyTypeMap: Record<string, string> = {
  alert: '预算预警',
  notice: '系统通知',
  approval: '审批消息',
}
export type NotifyType = 'alert' | 'notice' | 'approval'

/** 消息状态 */
export const NotifyStatusMap: Record<string, string> = {
  unread: '未读',
  read: '已读',
}
export type NotifyStatus = 'unread' | 'read'

/** 消息记录（对齐后端 NotificationResponse） */
export interface NotificationItem {
  id: number
  type: NotifyType
  title: string
  summary: string
  content: string
  status: NotifyStatus
  createdAt: string
}